user/tests/wiring/serial_loopback/serial.cpp

```diff
@@ -26,6 +26,29 @@
 #include "application.h"
 #include "unit-test/unit-test.h"
 
+#ifndef USE_BUFFER_SIZE
+#warning "Using default 64 byte buffer"
+#define USE_BUFFER_SIZE SERIAL_BUFFER_SIZE
+#else
+hal_usart_buffer_config_t __attribute__((weak)) acquireSerial1Buffer()
```

Comment:
`__attribute__((weak))` needs to be removed?

Reply:
:+1: fixed